### PR TITLE
changed text for editor

### PR DESCRIPTION
**Pull request by Lizzaran360**
toggle invasions button had a text that was too long. shortened ti.

### CODE REVIEW

File: src/translation/german.c [hunk 1]
```diff
@@ -119,7 +119,7 @@ static translation_string all_strings[] = {
     {TR_HOTKEY_ARROW_DOWN, "Runter"},
     {TR_HOTKEY_ARROW_LEFT, "Links"},
     {TR_HOTKEY_ARROW_RIGHT, "Rechts"},
-    {TR_HOTKEY_TOGGLE_FULLSCREEN, "Vollbild ein-/ausschalten"},
+    {TR_HOTKEY_TOGGLE_FULLSCREEN, "Vollbild ein-/ausschalten"}
```
Comment (UberWaffe) [inline src/translation/german.c:122]: Missing a `,` at the end.

**Review by UberWaffe — CHANGES_REQUESTED** [review]

File: src/translation/german.c (hunk 1 above)
Comment (Lizzaran360) [reply src/translation/german.c:122]: Why is it missing when I didn't touch that?
Or did I by mistake? Like it was highlighted even though I wasn't looking for that part?

**Comment by UberWaffe** [comment]
@crudelios I think is ready to merge? Should I do so?

**Comment by crudelios** [comment]
I'm closing this since this has already been added on #862

**Pull request closed without merging**